modules/distribution/resources/operation_policies/specifications/ccRewriteResourcePath_v1.json

```
@@ -0,0 +1,36 @@
+{
+  "category": "Mediation",
+  "name": "rewriteResourcePath",
+  "version": "v1",
+  "displayName": "Rewrite Resource Path",
+  "description": "This policy allows you to rewrite the resource path of a request",
+  "policyAttributes": [
+    {
+      "name": "newResourcePath",
+      "displayName": "New Resource Path",
+      "description": "Name of the new resource path",
+      "validationRegex": "^\\/?(({uri\\.var\\.[\\w\\/\\-]+})|[\\w\\/\\-\\s])+$",
```

Can we double-check these with the UI too? Looks like just `/` without the `\\` works fine. With this regex, the UI gets unexpected validation errors.

```suggestion
      "validationRegex": "^/?(({uri\\.var\\.[\\w\\/\\-]+})|[\\w\\/\\-\\s])+$",
```

---

using the same synapse policy, since it will support {uri.var.petID} format with the PR https://github.com/wso2/product-apim/pull/12670